tdrs-backend/tdpservice/search_indexes/migrations/0008_auto_20230424_1913.py

```diff
@@ -0,0 +1,451 @@
+# Generated by Django 3.2.15 on 2023-04-24 19:13
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('search_indexes', '0007_ssp_m1_ssp_m2_ssp_m3'),
+    ]
+
+    operations = [
```

Comment:
i'm getting a migration issue locally. `makemigrations` or `migrate` is prompting me for a field rename, so `docker-compose up` fails
```bash
Did you rename tanf_t2.aid_aged_blind to tanf_t2.AMOUNT_OF_EARNED_INCOME (a IntegerField)? [y/N]
```

i don't see any `RemoveField` for `aid_aged_blind`. anyway, i think it'd be nice to squash the migrations. in this case, we don't have any `t2/t3` documents we need to preserve so i think you could do the migration in two steps: `RemoveField` for all the old fields, then `AddField` for all the new ones. If delete all migrations, stash your changes, comment out all the old fields, that will allow you to generate the first migration. then you can unstash your changes and generate another.

hopefully that makes all this a little simpler and more predictable